ace-setup-mq/remove-resources.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Functions
+printUsage() {
+    echo -e "\033[1;33mThis script will delete the specified objects:\033[0m"
+    echo "1. Check user running this script has authority to execute MQSC commands"
+    echo "2. Remove the qmgr authority(connect+inq) for the specified user"
+    echo "3. Delete the specified listener/channel/topic"
+    echo -e "\033[1;33m Usage: $0 -q <QMGR_NAME> -d <MQ_BIN_PATH> [-u AUTH_USER] [-l LISTENER_NAME] [-c CHANNEL_NAME] [-t TOPIC_NAME]\033[0m"
+    echo -e "\033[1;33m Example: $0 -q QM1 -d /opt/mqm/bin -u root -l LS2 -c INSTANA.ACE.SVRCONN -t INSTANA.ACE.BROKER.TOPIC\033[0m"
+}
+
+# 1. Check the current user has authority to execute MQSC commands
+preCheck() {
+    echo "Check current user has the authority to execute MQSC commands"
+    if groups | grep -q '\bmqm\b'; then
+        echo -e "\033[1;32mINFO:The user launches the script belongs to mqm group\033[0m"
```

Comment:
```suggestion
        echo -e "\033[1;32mINFO:The user launched the script belongs to mqm group\033[0m"
```

Reply:
Optionally, t would be nice to have someone from ID to review the echo messages.